src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/VanillaMachinesListener.java

```diff
@@ -97,6 +97,25 @@ public void onAnvil(InventoryClickEvent e) {
         }
     }
 
+    @EventHandler(ignoreCancelled = true)
+    public void onCartographyTable(InventoryClickEvent e) {
+        // The Cartography Table was only ever added in MC 1.14
+        MinecraftVersion minecraftVersion = SlimefunPlugin.getMinecraftVersion();
```

Comment:
Why do you create a variable here? This just creates an unnecessary memory heap for gc to clean.

Reply:
There you go.